frontend/ui-pablo/app/components/Organisms/bonds/PageTitle.tsx

```diff
@@ -1,47 +1,61 @@
 import { BaseAsset, PairAsset } from "@/components/Atoms";
-import { getToken } from "@/defi/Tokens";
-import { TokenId } from "@/defi/types";
 import ArrowRightAltIcon from "@mui/icons-material/ArrowRightAlt";
-import { Box, Typography, useTheme, BoxProps } from "@mui/material";
+import { Box, Typography, BoxProps } from "@mui/material";
+import { MockedAsset } from "@/store/assets/assets.types";
+import { BondPrincipalAsset } from "@/defi/hooks/bonds/useBondOffers";
 
-export type PageTitleProps ={
-  tokenId1: TokenId;
-  tokenId2: TokenId;
+export type PageTitleProps = {
+  principalAsset: BondPrincipalAsset;
+  rewardAsset: MockedAsset | undefined;
   iconSize?: number;
 } & BoxProps;
 export const PageTitle: React.FC<PageTitleProps> = ({
-  tokenId1,
-  tokenId2,
+  principalAsset,
+  rewardAsset,
   iconSize = 67,
   ...boxProps
 }) => {
-  const token1 = getToken(tokenId1);
-  const token2 = getToken(tokenId2);
-  const pablo = getToken('pablo');
+
+  const { lpPrincipalAsset, simplePrincipalAsset } = principalAsset
+  const { baseAsset, quoteAsset } = lpPrincipalAsset;
+
   return (
     <Box width="100%" {...boxProps}>
-      <Box
-        display="flex"
-        justifyContent="center"
-        alignItems="center"
-        gap={3.5}
-      >
-        <PairAsset
-          assets={[
-            {icon: token1.icon, label: token1.symbol},
-            {icon: token2.icon, label: token2.symbol},
-          ]}
-          label={`LP ${token1.symbol}-${token2.symbol}`}
-          LabelProps={{variant: "h4"}}
-          iconSize={iconSize}
-        />
-        <ArrowRightAltIcon sx={{color: "text.secondary"}} />
-        <BaseAsset
-          icon={`/tokens/pablo_dark.svg`}
-          iconSize={67}
-          label={pablo.symbol}
-          LabelProps={{variant: "h4"}}
-        />
+      <Box display="flex" justifyContent="center" alignItems="center" gap={3.5}>
+        {baseAsset &&
+        quoteAsset ? (
```

Comment:
Same suggestion as in `bonds/SupplySummary.tsx` to avoid chaining ternary expressions

Reply:
updated